src/data_hub/api.py

```diff
@@ -239,48 +239,6 @@ def list_rows(
     )
 
 
-@beartype
-@ensure_client
-def download_file(
```

Comment:
Something to think about: if an existing code block uses this function, the user will not be able to re-run the code block without first updating to `download_files`. This may be especially confusing if AI wrote the code.

Reply:
hmm, yes, it's a breaking change for sure. good catch. i'll include a stub to use download_files under the hood

Reply:
actually changed my mind. the LLM can never generate this code because it doesn't know about this function